addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16541,3 +16541,13 @@ msgstr ""
 msgctxt "#38016"
 msgid "%d fps"
 msgstr ""
+
+#: system/settings/rbp.xml system/settings/imx6.xml
+msgctxt "#38120"
+msgid "Support 8 channel DTS HD audio decoding"
+msgstr ""
+
+#: system/settings/rbp.xml system/settings/imx6.xml
+msgctxt "#38121"
+msgid "This option supports 8 channel DTS HD decoding, but may use more CPU. It is only available when DTS and DTS-HD audio passthrough is disabled"
```

Comment:
We changed all audio related settings to be a bit more user friendly and also point out the benefit a user will have in a less technical way. So maybe we can rephrase this a bit as well, like pointing out that you're able to listen to DTS-HD even if your AVR doesn't support it? This would be my take on it, but feel free to change it:

"Enables decoding for DTS-HD audio streams. This allows conversion of DTS-HD audio into a format supported by the connected audio device and will give lossless 8 channel audio fidelity on devices that support PCM. Note that this might increase CPU load and is only available when DTS and DTS-HD audio passthrough is disabled."

PS: these labels are also used in pi.xml and pi2.xml files for which the useage comment is missing. A small contextual note would also be nice on both labels, like "Name of a setting" and "Description of setting #38120 "Foo-> Bar -> Enable DTS-HD audio decoding"